make_nf_run_script.py

```diff
@@ -0,0 +1,38 @@
+import os
+import argparse
+
+parser = argparse.ArgumentParser(description='Generate run script for nextflow pipelines')
+parser.add_argument('--project', required=True, help='Project name')
+parser.add_argument('--genome', required=True, help='Reference genome, e.g. GRCm38')
+parser.add_argument('--pipeline', required=True, help='Reference genome, e.g. methylseq')
+
+args = parser.parse_args()
+project = args.project
+genome = args.genome
+pipeline = args.pipeline
+
+analysis_path = "/proj/ngi2016001/nobackup/NGI/ANALYSIS/"
+scripts_path = os.path.join(analysis_path, project, "scripts")
+template_path = "/lupus/ngi/production/latest/sw/upps_standalone_scripts/run_script_templates"
+
+#Keep old template path for now in case not rolled out yet from github
```

Comment:
Since the config path in the rnaseq_GRCh38_gencode is hardcoded to the "standalone_scripts" path anyway, maybe this precaution can be removed?

Reply:
True, dropped it